docs/installation.md

```diff
@@ -0,0 +1,45 @@
+# Installation of RMT
+
+## Installation on SLE 15
+
+1. If your server isn't activated yet, activate it with the command `SUSEConnect -r <regcode>`.
+2. Activate the Server Applications Module for your version of SLE:
+    * SLE 15 SP2 - `SUSEConnect -p sle-module-server-applications/15.2/x86_64`
+    * SLE 15 SP1 - `SUSEConnect -p sle-module-server-applications/15.1/x86_64`
+    * SLE 15 - `SUSEConnect -p sle-module-server-applications/15/x86_64`
+3. Install RMT and its YaST installation wizard with the command `zypper in rmt-server yast2-rmt`.
```

Comment:
Links to SLE 15 iso might be very helpful.

Reply:
I added a note under `# Installation of RMT`